Frameworks/UIKit/UILabel.mm

```diff
@@ -44,61 +44,70 @@ @implementation UILabel {
     UILineBreakMode _lineBreakMode;
     BOOL _adjustFontSize;
     float _minimumFontSize;
-    float _originalFontSize;
     int _numberOfLines;
     BOOL _isDisabled;
     BOOL _isHighlighted;
     UIBaselineAdjustment _baselineAdjustment;
     StrongId<WXCTextBlock> _textBlock;
 }
 
-- (void)adjustFontSizeToFit {
+- (void)_adjustFontSizeToFit {
```

Comment:
Also, if you refactored this method a bit (to take starting/min font sizes and a frame as function arguments, you could return the final font to a calling function that applies it to _font and does the relayout.  At which point you could write a bunch of self-contained UTs against this method.

---
In reply to: [97400127](https://github.com/Microsoft/WinObjC/pull/1768#discussion_r97400127) [](ancestors = 97400127,97400039)